shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/main/java/org/apache/shardingsphere/test/sql/parser/parameterized/asserts/segment/expression/ExpressionAssert.java

```diff
@@ -65,7 +87,7 @@ public static void assertParameterMarkerExpression(final SQLCaseAssertContext as
      */
     public static void assertLiteralExpression(final SQLCaseAssertContext assertContext,
                                                 final LiteralExpressionSegment actual, final ExpectedLiteralExpression expected) {
-        assertNotNull(assertContext.getText("Expected literal expression should exist."));
+        assertNotNull(assertContext.getText("Expected literal expression should exist."), expected);
```

Comment:
@lwtdev Like this, maybe it's better to go to assert actual based on expect.

Reply:
@jingshanglu OK, I will modify it.